server/core/middleware/auth.middleware.ts

```diff
@@ -51,10 +63,15 @@ export async function getUserFromCookieIfExists(req: RequestWithPotentialUser) {
 
 		const verificationResponse = jwt.verify(cookies.Authorization, secret) as DataStoredInToken;
 		const userId = verificationResponse.id;
-		const user = await userService.getById(userId)
+		
+		try {
+			const user = await userService.getById(userId)
```

Comment:
Let's do this through `AuthService` as well.

Reply:
Did you mean `Users.findOne`? Or should I refactor some new method in `AuthService` which should fit this case in case you want the `auth.middleware` not having anything to do with `Users` directly?

Reply:
For now, I went with: https://github.com/mbrizic/ratethegame/pull/28/commits/b158c0718600a8adeb6bacdcd5d09c7e8ab913e4
Please comment if we should go with some other direction :D

Reply:
I'd hide all of that under some new `authService.getUserById` method. Main reason is that `findByPk` call will get dirtier when we add some database optimizations on it to only return fields that we need, which we definitely need to do someday, as the default query behaviour here is to join user with all its other tables (and we basically only need username and password here).

Reply:
Resolved in: https://github.com/mbrizic/ratethegame/pull/28/commits/146ea5f5f9e69a3b54e2d26ebef5aa414fcdd6ea